### PR TITLE
Update from v4.4.x to v4.5

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | N/A
| Versions      | all
| Edition       | all

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Redirects cover removed/moved pages
- [x] Code samples are working
- [x] PHP code samples have been fixed with PHP CS fixer
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

**Review by mnocon — COMMENTED** [review]
I've tested this guide together with https://github.com/ibexa/installer/pull/99/ and https://github.com/ibexa/installer/pull/101

1) There are new tables, available in the OSS version, that don't have an upgrade script (ibexa_token and ibexa_token_type)
Added in https://github.com/ibexa/core/pull/217

2) There's no mention of the Shipment and Order policies added by default

I think these two migrations are responsible for them:
https://github.com/ibexa/shipping/blob/main/src/bundle/Resources/install/migrations/shipment_permissions.yaml
https://github.com/ibexa/order-management/blob/main/src/bundle/Resources/install/migrations/order_permissions.yaml

3) There are differences in database schema between normal installation and an upgrade

a) ibexa_payment table:
in our schema the owner field is nullable: https://github.com/ibexa/payment/blob/main/src/bundle/Resources/config/schema.yaml#L121-L123
in the script it's not: https://github.com/ibexa/installer/pull/101/files#diff-3acc61ccaf8c51bec0207363c084661b3c914faf7facef830a3402512dc7a557R418

b) ibexa_order table:
the identifier column does not have the `uuid` type after the upgrade
the lenght of the source column has different lenghts (190 vs 255)

https://github.com/ibexa/order-management/blob/main/src/bundle/Resources/config/schema.yaml#L10-L26
https://github.com/ibexa/installer/blob/5416-added-shipping-upgrade-scripts/upgrade/db/postgresql/commerce/ibexa-4.4.latest-to-4.5.0.sql#L9

left side - upgraded
right - reference 4.5
obraz

File: docs/update_and_migration/from_4.4/update_from_4.4.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+- Customer Portal:
+
+```bash
+php bin/console ibexa:migrations:import vendor/ibexa/corporate-account/src/bundle/Resources/migrations/customer_portal.yaml --name=2023_03_06_13_00_customer_portal.yaml
+php bin/console ibexa:migration:migrate --file=2023_03_06_13_00_customer_portal.yaml
+```
+
+- Corporate account — this migration allows all company members to shop in the frontend shop. If you have implemented business logic that depends on keeping company members out of the frontend shop, you can skip it:
+
+```bash
```
[the remark this answers is not in the document]
Comment (MagdalenaZuba) [reply docs/update_and_migration/from_4.4/update_from_4.4.md:113]: Masz racje PR jeszcze czeka na merge: https://github.com/ibexa/storefront/pull/100

**Comment by konradoboza** [comment]
>There are differences in database schema between normal installation and an upgrade

This should be fixed via:
https://github.com/ibexa/installer/pull/99/commits/d11557bcbf5713294481c4bb7e8eee28fb734694
https://github.com/ibexa/installer/pull/101/commits/734c1295e1f96bc9a61eb843bf20e69876a18a9b

**Comment by ciastektk** [comment]
> I've tested this guide together with [ibexa/installer#99](https://github.com/ibexa/installer/pull/99) and [ibexa/installer#101](https://github.com/ibexa/installer/pull/101)
> 
> 1. There are new tables, available in the OSS version, that don't have an upgrade script (ibexa_token and ibexa_token_type)
>    Added in [IBX-5314: Implemented token storage ibexa/core#217](https://github.com/ibexa/core/pull/217)

Upgrade scripts for token tables: https://github.com/ibexa/installer/pull/105

**Review by mnocon — APPROVED** [review]
It works! 🎉

**Review by DominikaK — APPROVED** [review]

**Pull request merged**